docs/modules_contentobject.md

```diff
@@ -148,3 +151,21 @@ Removes a translation from the content object
 $ eep contentobject translationremove <object id> <locale>
 ```
 
+## stateassignbyid
+Assigns an object state by state id
+```sh
+eep contentobject stateassignbyid <object id> <state id>
+```
+
+## stateassignbyidentifier
+Assigns an object state by state/group identifier e.g. ez_lock/locked
```

Comment:
Could you make at a suggestion for at least one location where a person would find some of these identifiers, like an INI or code-file? Doesn't have to be an anything more than "See, for example, settings/states.ini" or similar.

Reply:
Good idea. Doc update has been added.